### DISCUSSION TITLE
No whatsapp option available

### DISCUSSION THREAD

**Discussion by ural89** [d99kris/nchat#266] [Q&A]
I have manually build it from source in Linux Mint. I dont have option for WhatsApp.
![image](https://github.com/user-attachments/assets/54deede9-06a3-4a26-b988-f57a3cd8aee2)

**Comment by d99kris** [comment]
Hi @ural89 - can I check if you have `go` installed, and which version? You can check with `go version`.

The whatsapp protocol implementation in nchat requires v1.21 or later. If your version is older, try get a newer `go` version, and then do a clean rebuild of nchat.

**Reply by ural89** [reply]
thanks for your reply! I have downloaded this version: https://github.com/d99kris/nchat.git 
Which one is the go version please?

**Reply by d99kris** [reply]
In your terminal/shell please type this command:

    go version

and share the output here, then I will see what would be the next step.

**Reply by ural89** [reply]
Ah! Sorry, I misunderstood. It is fixed now when I upgraded the go. Thanks!
